CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # Contributing to gbasis
 Hello! Thank you for taking your time to contribute!
 
-Please note we have a [Code of Conduct](CODE_OF_CONDUCT.md) , please follow it in all your interactions with the project. Please report unacceptable behavior to [theochem@github.com](mailto:theochem@github.com).
+Please note we have a [Code of Conduct](https://qcdevs.org/guidelines/QCDevsCodeOfConduct/) , please follow it in all your interactions with the project. Please report unacceptable behavior to [theochem@github.com](mailto:theochem@github.com).
```

Comment:
@PaulWAyers, should we also change the email to `qcdevs` email instead of `theochem`? I am not sure which one is being actively monitored.

Reply:
They forward to each other, so it's fine. But I was thinking to update the web site one too, and also use the `paul@qcdevs.org`, `farnaz@qcdevs.org`, etc. for the admin teams e-mails too.